pkgs/by-name/tl/tlock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-O6erxzanSO5BjMnSSmn89w9SA+xyHhp0SSDkCk5hp8Y=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-G402CigSvloF/SI9Wbcts/So1impMUH5kroxDD/KKew=";
+
+  doCheck = false;
+
+  excludedPackages = [
+    "bubbletea"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/eklairs/tlock/tlock-internal/constants.VERSION=v${version}"
+  ];
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
And edit the other attributes appropriately. (e.g. `licenses` -> `lib.licenses`)